### DISCUSSION TITLE
No 'porosity' argument in cylinder generator

### DISCUSSION THREAD

**Discussion by taylrcawte** [PMEAL/porespy#487] [Q&A] [answered]
Hello, 

i was looking to generate fibrous mats using the porespy cylinder generator; in the document guide there is a 'porosity' argument in the generator function, however when i download using ```conda install -c conda-forge porespy``` the generator does not have this argument. 

Should i use a different version of porespy? 

Thanks in advance

**Comment by jgostick** [comment, marked as answer]
we're in the midst of a major overhaul to move to V2. It will get sorted out within the next week or so.

**Reply by taylrcawte** [reply]
Thanks!

**Reply by jgostick** [reply]
I just want to confirm that this option is on dev so will be part of the main release soon.
